README.md

```diff
@@ -88,11 +88,31 @@ deploy.sh
 ```
 
 # Deploy Cloud Object Storage Package with IBM Cloud Console
-Currently there is no direct way to deploy the Cloud Object Storage package from the IBM Cloud Functions UI.  However, there is a workaround that provides you with a way to install the cloud-object-storage package in the NodeJS runtime. 
 
-To do this you will have to install the **Upload Image** template which installs the **cloud-object-storage** package as a prerequisite for the template. 
+## Deploy Package from the UI
+In the Cloud Functions console, go to the Create page External link icon.
 
-To do this follow the installation process for the UI on the [Upload Image template's github repo](https://github.com/ibm-functions/template-cloud-object-storage#deploying-from-the-ibm-cloud-functions-ui)
+Using the Cloud Foundry Org and Cloud Foundry Space lists, select the namespace that you want to install the Object Storage package into. Namespaces are formed from the combined org and space names.
+
+Click Install Packages.
+
+Click on the IBM Cloud Object Storage Package group, and then click on the IBM Cloud Object Storage Package.
+
+In the Available Runtimes section, select either NodeJS or Python from the drop-down list and then click Install.
+
+Once the Package has been installed you will be redirected to the Actions page and can search for your new Package, which is named cloud-object-storage.
+
+##Bind Service Credentials
+
+To use the Actions in the cloud-object-storage Package, you must bind service credentials to the actions.
+
+To bind service credentials to all actions in the package, follow steps 5 and 6 in the CLI instructions listed above.
+
+To bind service credentials to individual actions, complete the following steps in the UI. Note: You must complete the following steps for each action that you want to use.
```

Comment:
I would combine the three sentences in lines 107 through 111 into one sentence:

To use the actions in the cloud-object-storage package, you must bind service credentials to the actions.  Note: You must complete the following steps for each action that you want to use.